components/RainbowKit.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRouter } from "next/router";
+import "@rainbow-me/rainbowkit/styles.css";
+import { configureChains, WagmiConfig, createConfig } from 'wagmi';
+import {
+    darkTheme,
+    connectorsForWallets,
+    RainbowKitProvider
+} from '@rainbow-me/rainbowkit';
+const WALLETCONNECT_PROJECT_ID = '28168903b2d30c75e5f7f2d71902581b';
+import { publicProvider } from 'wagmi/providers/public';
+import { walletConnectWallet, rainbowWallet, metaMaskWallet, coinbaseWallet, bitKeepWallet, argentWallet } from '@rainbow-me/rainbowkit/wallets';
+import { useSettingsState } from "../context/settings";
+
+type Props = {
+    children: JSX.Element | JSX.Element[]
+}
+
```

Comment:
We can remove the chainconfigs file now, right?

Reply:
removed